packages/top-app-bar/index.tsx

```diff
@@ -219,6 +219,33 @@ class TopAppBar<T extends HTMLElement = HTMLHeadingElement> extends React.Compon
         }
         return 0;
       },
+      registerResizeHandler: (handler: SpecificEventListener<'resize'>) => {
+        window.addEventListener('resize', handler);
+      },
+      deregisterResizeHandler: (handler: SpecificEventListener<'resize'>) => {
+        window.removeEventListener('resize', handler);
+      },
+      registerNavigationIconInteractionHandler: <K extends EventType>(type: K, handler: SpecificEventListener<K>) => {
```

Comment:
For these three adapter methods regarding navigationIcon, can we not implement them (or just have them as empty bodies? And just add a comment linking to this issue: https://github.com/material-components/material-components-web/issues/2813.

No one using the components should need to do this, since they can just add an `onClick` method to the Icon.